src/Control/Monad/Free/Trans.purs

```diff
@@ -76,6 +79,12 @@ instance monadRecFreeT :: (Functor f, Monad m) => MonadRec (FreeT f m) where
         Loop s1 -> go s1
         Done a -> pure a
 
+instance semigroupFreeT :: (Functor f, Monad m, Monoid w) => Semigroup (FreeT f m w) where
+  append = lift2 append
+
+instance monoidFreeT :: (Functor f, Monad m, Monoid w) => Monoid (FreeT f m w) where
+  mempty = pure mempty
```

Comment:
Actually I'm not sure about that - FreeT's `Apply` instance depends on `Monad` due to the use of `ap` - I'm not sure if there's a way around that, I'm not too familiar with FreeT's design :x